localstack/services/awslambda/lambda_api.py

```diff
@@ -1375,6 +1493,122 @@ def add_permission(function):
     return result
 
 
+@app.route("%s/functions/<function>/url" % API_PATH_ROOT_2, methods=["POST"])
```

Comment:
These functions newly added to the "legacy" lambda provider, but are not implemented in the "new / ASF" lambda provider (`provider.py` in the same package).
@dominikschubert Is there a pattern on how you are currently handling the sync between the two implementations? Do you try to keep them in sync with every PR or will there be another iteration to add new changes in the legacy providers to the new provider later on?

Reply:
@dominikschubert let me know if the code needs to be moved.

Reply:
It is ok to be here for now, we will merge it in the works of the new provider, once necessary